docs/advanced-guides/creating-plugins.md

```diff
@@ -43,9 +43,12 @@ The `index.js` is the entrypoint for your plugin and provides the add/remove fun
 
 1. Change the `NPM_MODULE_NAME` to `react-native-radio-buttons`
 2. Since this npm package does not require a linking step, we will change `ignite.addModule` to have `link: false` and `ignite.removeModule` to have `unlink: false`
+3. Make sure `EXAMPLE_FILE` matches the filename under `templates`
```

Comment:
Note: This advice really only applies once https://github.com/infinitered/ignite/issues/690 is fixed. Currently, you actually don't want them to necessarily match, because we are appending `.ejs` to the end of `EXAMPLE_FILE` in `ignite/packages/ignite/src/extensions/ignite.js:201` as part of `addComponentExample`.